src/vivarium_gates_mncnh/components/lbwsg.py

```diff
@@ -199,18 +204,23 @@ def get_relative_risk_for_age_group(age_group: str) -> pd.Series:
         self.population_view.update(pd.concat(relative_risk_columns, axis=1))
 
 
-class LBWSGPAFCalculationRiskEffect(LBWSGRiskEffect_):
+####################################
+# LBWSG PAF Calculation Components #
+####################################
+
+
+class LBWSGPAFCalculationRiskEffect(LBWSGRiskEffect):
```

Comment:
Im inheriting from the model version of the VPH components to more easily handle the column differences.